dts/riscv/andes/andes_v5_ae350.dtsi

```diff
@@ -204,8 +204,8 @@
 					       &CPU2_intc 7 &CPU3_intc 7>;
 		};
 
-		smu: smu@f0100000 {
-			compatible = "andestech,atcsmu100";
```

Comment:
Out of curiosity - is this binding used anywhere else? If not, why are we not removing it?

Reply:
No, we didn't use this binding and didn't add `andestech,atcsmu100.yaml` either.